mkosi.extra/etc/systemd/journal.conf.d/augment-journald.conf

```diff
@@ -7,5 +7,5 @@ RateLimitIntervalSec=0
 RateLimitBurst=0
 RuntimeKeepFree=5%
 ForwardToConsole=true
-TTYPath=/dev/tty1
+TTYPath=/dev/ttyS0
```

Comment:
Is this only viable for testing? What about actual systems in hardware?

Reply:
To my knowledge, we have configured all systems to provide SOL, so the log should be available there too.
But I can also revert it back.

Reply:
I've changed it back.